### PR TITLE
Add travisci file and status image.

### PR DESCRIPTION
**Pull request by tmcgilchrist**
https://github.com/ambiata/introduction-to-fp-in-scala/issues/15

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: scala
+scala:
+  - 2.10.3
+
+jdk:
+  - oraclejdk7
+  - openjdk6
+
+script: sbt ++$TRAVIS_SCALA_VERSION "test:compile"
```
Comment (charleso) [inline .travis.yml]: I'm assuming if you don't add this line the tests run as well? Do they fail?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: scala
+scala:
+  - 2.10.3
+
+jdk:
+  - oraclejdk7
+  - openjdk6
```
Comment (charleso) [inline .travis.yml]: Probably don't need both, maybe just `openjdk6`?

File: .travis.yml (hunk 1 above)
Comment (markhibberd) [inline .travis.yml]: Yes they are incomplete.

File: README.md [hunk 3]
```diff
@@ -1,5 +1,4 @@
-# Introduction to FP in Scala
-
+# Introduction to FP in Scala [![Build Status](https://travis-ci.org/tmcgilchrist/introduction-to-fp-in-scala.svg?branch=travis_ci)](https://travis-ci.org/tmcgilchrist/introduction-to-fp-in-scala)
```
Comment (markhibberd) [inline README.md]: Can you change these urls to ambiata and the branch to master, then looks good. Thanks.

**Comment by charleso** [comment]
:+1: Thanks @tmcgilchrist

File: .travis.yml (hunk 2 above)
Comment (tmcgilchrist) [inline .travis.yml]: Sure.

File: README.md (hunk 3 above)
Comment (tmcgilchrist) [inline README.md]: Done.

**Comment by markhibberd** [comment]
@tmcgilchrist Thanks.

**Pull request merged**